packages/opentelemetry-api/README.md

```diff
@@ -9,13 +9,14 @@ This package provides everything needed to interact with the OpenTelemetry API,
 
 ## Quick Start
 
-To get started tracing you need to install the SDK and plugins, create a TracerProvider, and register it with the API.
+To get started you need to install the SDK and plugins, create a TracerProvider and/or MeterProvider, and register it with the API.
 
 ### Install Dependencies
 
 ```sh
 $ # Install tracing dependencies
 $ npm install \
+    @opentelemetry/api \
     @opentelemetry/core \
     @opentelemetry/node \
```

Comment:
I think it might be worth to mention here that this example is for node only - there were already cases when people tried to use Jaeger in browser

Reply:
done.